synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc.sh

```diff
@@ -59,7 +59,8 @@ python3 -m docuploader upload . \
 
 popd
 
-# V2
+# V2 due to problems w/ the released javadoc plugin doclava, Java 8 is required.  Beware of accidental updates.
```

Comment:
should we be locking the version of the plugin in the pom.xml then? 

Is "doclava" correct?

Reply:
It's done.  DocLava hasn't been published (by us) in 6 years.  I'm trying to get this released so we can the CloudRAD done.  We are also working on a replacement.

Reply:
OK. I just never encountered that before. I'm still having trouble parsing this comment. It might be worth rewriting.